test/e2e/compare.test.ts

```diff
@@ -13,64 +13,52 @@ const CODE_PROJECT_2_WARNING = `${CODE_PATH}/Test2Warning.tsx`;
 const CODE_PROJECT_3 = `${CODE_PATH}/Test3Mixed.tsx`;
 
 it('says projects are in sync when they do match', async () => {
-  const out = await run([
-    'compare',
-    '--api-key',
-    PROJECT_PAK_2,
-    '--extractor',
-    'react',
-    CODE_PROJECT_2_COMPLETE,
-  ]);
+  const out = await run(
+    ['compare', '--api-key', PROJECT_PAK_2, CODE_PROJECT_2_COMPLETE],
+    undefined,
+    20e3
+  );
 
   expect(out.code).toBe(0);
   expect(out.stdout).toContain('is in sync');
-});
+}, 30e3);
 
 it('detects new keys in code projects', async () => {
-  const out = await run([
-    'compare',
-    '--api-key',
-    PROJECT_PAK_2,
-    '--extractor',
-    'react',
-    CODE_PROJECT_2_ADDED,
-  ]);
+  const out = await run(
+    ['compare', '--api-key', PROJECT_PAK_2, CODE_PROJECT_2_ADDED],
+    undefined,
+    20e3
+  );
 
   expect(out.code).toBe(0);
   expect(out.stdout).toContain('out of sync');
   expect(out.stdout).toContain('2 new strings');
   // Matching \n is important to ensure it properly understood it is not a namespaced string
   expect(out.stdout).toContain('+ mouse-name\n');
   expect(out.stdout).toContain('+ mouse-sound\n');
-});
+}, 30e3);
 
 it('detects keys that no longer exist', async () => {
-  const out = await run([
-    'compare',
-    '--api-key',
-    PROJECT_PAK_2,
-    '--extractor',
-    'react',
-    CODE_PROJECT_2_DELETED,
-  ]);
+  const out = await run(
+    ['compare', '--api-key', PROJECT_PAK_2, CODE_PROJECT_2_DELETED],
+    undefined,
+    20e3
```

Comment:
What are these magic constants?

Reply:
Could we get rid of them?

Reply:
it's the timeout, I could make it more explicit but I just did what Jest did as well with how it deals with timeout. Because the extractor is heavy, on GHA it tends to timeout so I give it more time here

Reply:
Ah, ok. I guess thats fine then